examples/docs_beta_snippets/docs_beta_snippets_tests/snippet_checks/guides/components/test_components_docs.py

```diff
@@ -282,7 +282,13 @@ def next_snip_no():
     project_dir: ../../../dbt/jdbt
   asset_attributes:
     key: "target/main/{{ node.name }}"
-    """,
+""",
+            )
+            run_command_and_snippet_output(
+                cmd="dg component check --disable-cache --verbose --no-use-dg-managed-environment",
```

Comment:
This ensures we test against the schema found in dev dagster, rather than the published modules. This is important in this case because this PR is stacked on a dagster-dbt schema fix which isn't published yet.